src/Component/RuleTable/RuleTable.tsx

```diff
@@ -114,13 +123,25 @@ export class RuleTable extends React.Component<RuleTableProps, RuleTableState> {
   }
 
   symbolizerRenderer = (text: string, record: RuleRecord) => {
+    const {
+      rendererType,
+      sldRendererProps
+    } = this.props;
+
+    const onSymbolizerClick = (symbolizers: Symbolizer[], event: any) => {
+      const filterPosition = event.target.getBoundingClientRect();
+      this.onSymbolizerClick(record, filterPosition);
+    };
+
     return (
-      <Renderer
+      rendererType === 'SLD' ? <SLDRenderer
+      symbolizers={record.symbolizers}
+      onClick={onSymbolizerClick}
+      {...sldRendererProps}
+    />
+      : <Renderer
```

Comment:
Minor: Could you add brackets arount the Element, to make sure the whole element will be read?
Example: 
```
rendererType === 'SLD' ? (<SldRenderer
                             ...props
                           />) : ....
```

Reply:
:white_check_mark: , see fbe63e9